yarn.lock

```diff
@@ -1608,6 +1608,15 @@
     classnames "^2.2.6"
     tslib "^1.10.0"
 
+"@fluentui/react-focus-management@^9.0.0-alpha.12":
```

Comment:
Can you please run `yarn syncpack:fix` & `yarn` locally? This change is not expected IMO.